apps/api-extractor-model/src/model/ApiPackage.ts

```diff
@@ -14,6 +14,18 @@ import { ApiDocumentedItem, IApiDocumentedItemOptions } from '../items/ApiDocume
 import { ApiEntryPoint } from './ApiEntryPoint';
 import { IApiNameMixinOptions, ApiNameMixin } from '../mixins/ApiNameMixin';
 import { DeserializerContext, ApiJsonSchemaVersion } from './DeserializerContext';
+import { TSDocConfiguration, TSDocTagDefinition, TSDocTagSyntaxKind } from '@microsoft/tsdoc';
+
+interface ITagConfigJson {
```

Comment:
@octogonz defining these here is odd to me because they're essentially a copy of the interfaces defined in `tsdoc-config` and are the product of `saveToObject()`, but `saveToObject()` returns `unknown`. Should I do something to address this?

Reply:
It was easy, so I went ahead and implemented the missing `loadFromObject()` operation in https://github.com/microsoft/tsdoc/pull/288